rosdep/python.yaml

```diff
@@ -3132,6 +3132,16 @@ python-open3d-pip:
   ubuntu:
     pip:
       packages: [open3d-python]
+python3-openai-pip:
```

Comment:
Oh, right.  This key also needs to move to be among the `python3-` section, otherwise it is out-of-order alphabetically.